packages/stream/package.json

```diff
@@ -24,7 +24,10 @@
   "exports": {
     ".": {
       "types": "./src/lib.d.ts",
-      "browser": "./src/lib.js",
+      "browser": {
+        "require": "./src/stream.cjs",
```

Comment:
Curious why this is the only one not named `lib`?

Reply:
oh I see it's just a direct export of another lib so we don't really need a build version for it?